Update benchmark.md (#104)
JuliaSmoothOptimizers/OptimizationProblems.jl: docs/src/benchmark.md

``` @example ex1
problems = (MathOptNLPModel(eval(problem)(), name=string(problem)) for problem ∈ filter(x -> x != :PureJuMP, names(OptimizationProblems.PureJuMP)))
```

> Obviously, the same could be achieved using `OptimizationProblems.ADNLPProblems`. For example, you can write the following if you want probelms from `OptimizationProblems.ADNLPProblems`:

Reviewer: ```suggestion
The same can be achieved using `OptimizationProblems.ADNLPProblems` as follows:
```
